client-ts/Microsoft.AspNetCore.SignalR.Client.TS.Tests/Connection.spec.ts

```diff
@@ -112,10 +112,10 @@ describe("Connection", () => {
             }
         } as IHttpConnectionOptions;
 
-        var connection = new HttpConnection("http://tempuri.org", options);
+        let connection = new HttpConnection("http://tempuri.org", options);
```

Comment:
I am leaving it as it is then. If we decide to change it to const we would probably have to change almost all variables which is out of scope for this PR. I was just annoyed by mixing `let` and `var` without any reason.